### DISCUSSION TITLE
从Windows向IOS端发送文件时提示“远程计算机网络拒绝访问”

### DISCUSSION THREAD

**Discussion by beker888** [localsend/localsend#672] [L-简体中文]
经测试，不知道是不是因为ios后台的墓碑机制的原因，打开ios端的LocalSend之后，如果是过了一段时间，再从Windows端传文件到手机，会提示“远程计算机网络拒绝访问”。在IOS端的localsend中重启服务器会解决这个问题，但是如果每次都是这样的话，会很不方便。不知道后续可有办法改进，让其随时可以传说连接。谢谢！

Localsend真是太棒了，个人非常喜欢！

**Comment by Yun1Mu** [comment]
我也有同样的问题

**Comment by ryusei233** [comment]
我也遇到了同样的问题，ios传win没事，win传ios必须在ios杀死再重启Localsend才行

**Comment by beker888** [comment]
> 我也遇到了同样的问题，ios传win没事，win传ios必须在ios杀死再重启Localsend才行
谢谢，明白了！

**Comment by magiczero** [comment]
今天遇到了，确实是ios机制问题，在设置里的服务器选项点一下重启就可以收到了

**Comment by kakei777** [comment]
同样的问题，解决明白了感谢。

**Comment by yubanmayi** [comment]
这个只要ios息屏后就会出现，可能是ios机制，重启服务器后就可以了

**Reply by Ghost-Sam1222** [reply]
我发现并不完全是，只要不是在前台都有可能会出现这个情况，iOS端尤为明显后台挂一会就不能传文件了。iPadOS端小标签化放一会也是如此

**Comment by sthfake** [comment]
一开始就传不成功，只能ios向外传
